src/include/units/quantity_point.h

```diff
@@ -0,0 +1,237 @@
+
+// The MIT License (MIT)
+//
+// Copyright (c) 2018 Mateusz Pusz
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#pragma once
+
+#include <units/quantity.h>
+
+namespace units {
+
+/**
+ * @brief A quantity point
+ *
+ * Property of a phenomenon, body, or substance, where the property has a magnitude that can be
+ * expressed by means of a number and a measurement unit.
```

Comment:
This is the same definition for `quantity`. How should it be amended?

Reply:
I am not sure what you mean here by "amended"?

Anyway, in my `quantity_point` type I was working on I had an additional last parameter `typename Traits = quantity_point_traits<U>`. So you can i.e. define two quantity_points of length working with different zero points (i.e. every airfield has a different altitude so HAGL is different for each of them but still we should be able to use them in one equation as each of them is defined for the same physical dimension).

Reply:
It's just that I copied the definition from `quantity`, so I thought this should be changed to account for the differences between it and a `quantity_point`.